strapi_to_mdx_converter.py

```diff
@@ -53,14 +48,21 @@ def save_mdx_files(slug_filename, json_data, lang_flag=False):
 
 
 def main():
-    paths = rq.get(PATHS_URL).json()
-    all_pages = get_all_pages(paths)
+    if rq.get(PATHS_URL).ok:
+        paths = rq.get(PATHS_URL).json()
+
+    all_pages = [i.get('slug') for i in paths]
 
     for page in tqdm(all_pages):
-        slug_filename = page.replace("/", "")
-        json_page_data = json.loads(rq.get(f"{CONTENT_PAGE_URL}{page}").text)
-        save_mdx_files(slug_filename, json_page_data)
-        save_mdx_files(slug_filename, json_page_data, True)
+        slug = page.replace("/", "")
+
+        try:
+            json_page_data = rq.get(f"{CONTENT_PAGE_URL}/{slug}").json()
```

Comment:
what are you trying to reach by removing slash at 57 and adding it here?

Reply:
removed variable that I use only once